requirements/base.txt

```diff
@@ -1,19 +1,19 @@
 chevron~=0.12
-click~=7.0
-Flask~=1.0.2
+click~=7.1
+Flask~=1.1.2
 #Need to add Schemas latest SDK.
 boto3~=1.14.0, >=1.14.23
-jmespath~=0.9.5
-PyYAML~=5.1
-cookiecutter~=1.6.0
+jmespath~=0.10.0
+PyYAML~=5.3
+cookiecutter~=1.7.2
 aws-sam-translator==1.27.0
 #docker minor version updates can include breaking changes. Auto update micro version only.
 docker~=4.2.0
 dateparser~=0.7
 python-dateutil~=2.6, <2.8.1
 requests==2.23.0
-serverlessrepo==0.1.9
+serverlessrepo==0.1.10
 aws_lambda_builders==1.1.0
-tomlkit==0.5.8
+tomlkit==0.7.0
```

Comment:
can we verify that that this does not break integ tests, we have to be careful with any library we upgrade, esp one which deals with configuration files.

Reply:
I keep getting this timeout every time I ran integration test locally:

```
tests/integration/local/invoke/test_integrations_cli.py::TestSamPython36HelloWorldIntegration::test_skip_pull_image_in_env_var
+++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++

~~~~~~~~~~~~~~~~~~~~~~~ Stack of MainThread (4356918720) ~~~~~~~~~~~~~~~~~~~~~~~
  File "/Users/xinhol/.pyenv/versions/samcli377/bin/pytest", line 10, in <module>
    sys.exit(console_main())
  File "/Users/xinhol/.pyenv/versions/3.7.7/envs/samcli377/lib/python3.7/site-packages/_pytest/config/__init__.py", line 187, in console_main
    code = main()
  File "/Users/xinhol/.pyenv/versions/3.7.7/envs/samcli377/lib/python3.7/site-packages/_pytest/config/__init__.py", line 165, in main
    config=config
  File "/Users/xinhol/.pyenv/versions/3.7.7/envs/samcli377/lib/python3.7/site-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/Users/xinhol/.pyenv/versions/3.7.7/envs/samcli377/lib/python3.7/site-packages/pluggy/manager.py", line 93, in _hookexec
```

So I decided to explore ways to run integration test on AppVeyor, and I notice `` is already checked in "continuous-integration/appveyor/pr/make-pr" why we still need to verify integration test again when CI already run the integration tests?

https://ci.appveyor.com/project/AWSSAMCLI/aws-sam-cli/builds/35969063/job/9apg0g6e2t0oacbc?fullLog=true#L4975

Reply:
Tested against the production sam-cli with:
- production generate a config that matches this PR's generated one
- this PR can load production generated config without correct stdout